inscriptions/ISic000510.xml

```diff
@@ -188,14 +193,31 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la">
+                <ab>
+                    <lb n="1"/><expan><abbr>Imp</abbr><ex>erator</ex></expan> <g ref="#interpunct">·</g> <name>Caesari</name>
+                    <lb n="2"/><expan><abbr>d</abbr><ex>omino</ex></expan> <expan><abbr>n</abbr><ex>ostro</ex></expan> <name><expan><abbr>Fl</abbr><ex>avio</ex></expan></name> <name><expan><abbr>Valenti</abbr><ex>niano</ex></expan></name>
+                    <lb n="3"/><name>Pio</name> <name>Felici</name> semper
+                    <lb n="4"/><name>Augusto</name>
+                    <lb n="5"/><name><expan><abbr>M</abbr></expan><ex>arcus</ex></name> <g ref="#interpunct">·</g> <name>Valerius</name>
+                    <lb n="6"/><name>Quintianus</name>
+                    <lb n="7"/><expan><abbr>v</abbr><ex>ir</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>c</abbr><ex>larissimus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>cons</abbr><ex>ularis</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>p</abbr><ex>rovinciae</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>S</abbr><ex>icilae</ex></expan>
+                    <lb n="8"/>clementiae
+                    <lb n="9"/>pietatisque eius
+                    <lb n="10"/>semper dicatis
+                    <lb n="11" break="no"/>simus
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of CIL</note></app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#GC">
+                <p>To the emperor Caesar, our master Flavius Valentinanus, pious, fortunate, always Augustus; Marcus Valerius Quintianus, very illustrious man, governor of the province of Sicilia, most consacrated to his clemency and piety.</p>
```

Comment:
To the emperor Caesar, our lord Flavius Valens, pious, fortunate, forever Augustus; Marcus Valerius Quintianus, vir clarissimus, governor (consularis) of the province of Sicily, forever most dedicated to his clemency and piety.